src/biogeophys/OzoneMod.F90

```diff
@@ -494,21 +640,21 @@ subroutine CalcOzoneUptakeOnePoint( &
        endif
 
        if (pftcon%evergreen(pft_type) == 1) then
-          leafturn = 1._r8/(pftcon%leaf_long(pft_type)*365._r8*24._r8)
+          leafturn = 1._r8/(pftcon%leaf_long(pft_type)*avg_dayspyr*secspday)
```

Comment:
I very much like using functions and constants to get things like the number of days in a year, and the seconds per day. That's an important improvement.

Reply:
thanks